src/commands/up.rs

```diff
@@ -387,25 +413,46 @@ impl UpCommand {
 }
 
 #[cfg(windows)]
-fn set_kill_on_ctrl_c(trigger_processes: &Vec<tokio::process::Child>) -> Result<(), anyhow::Error> {
+fn set_kill_on_ctrl_c(_pids: &[usize]) -> Result<(), anyhow::Error> {
     Ok(())
 }
 
 #[cfg(not(windows))]
-fn set_kill_on_ctrl_c(trigger_processes: &[tokio::process::Child]) -> Result<(), anyhow::Error> {
+fn set_kill_on_ctrl_c(pids: &[nix::unistd::Pid]) -> Result<(), anyhow::Error> {
+    let pids = pids.to_owned();
+    ctrlc::set_handler(move || {
+        kill_them_all_god_will_know_his_own(&pids);
+    })?;
+    Ok(())
+}
+
+#[cfg(windows)]
+fn get_pids(_trigger_processes: &[tokio::process::Child]) -> Vec<usize> {
+    vec![]
```

Comment:
I'm confused why this would be correct.

Reply:
_chuckle_ It's not "correct".  This PR tracks the original Ctrl+C propagation code, which was cfg-ed out on Windows (because the signals stuff doesn't have an equivalent); so although I need to have a function here (and could indeed get OS-level process handles) I can't do anything meaningful with them, at least not without a lot more research than "preserve what the old code did".

I can look at implementing Ctrl+C propagation on Windows for sure.  But I'd be inclined to look at a slightly different design for that, with processes starting suspended and being placed in a job object.  That's more than I want to bite off in this PR though...!